kuronometer-core/src/main/scala/com/github/pedrovgs/kuronometer/free/interpreter/csv/CsvReporter.scala

```diff
@@ -0,0 +1,99 @@
+package com.github.pedrovgs.kuronometer.free.interpreter.csv
+
+import java.io.{File, FileReader, FileWriter}
+
+import com.github.pedrovgs.kuronometer.KuronometerResults.KuronometerResult
+import com.github.pedrovgs.kuronometer.free.domain._
+import org.supercsv.io.{CsvBeanReader, CsvBeanWriter, ICsvBeanWriter}
+import org.supercsv.prefs.CsvPreference
+
+import scala.collection.mutable
+import scala.collection.mutable.ListBuffer
+
+class CsvReporter(implicit csvReporterConfig: CsvReporterConfig) {
```

Comment:
It depends on the granularity we want to provide. Do we need to abstract the usage of the library creating an algebra on top of a CSV library? If I'm not wrong, for this project, abstracting there is a remote or local reporter is enough. If in the future we need to write business logic rules around the CSV usage we will write an algebra 😃